src/main/java/core/basesyntax/BallCollor.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallCollor {
```

Comment:
```suggestion
public enum Color {
```

Don't need to specify that color if related to ball